README.md

```diff
@@ -1 +1,38 @@
-# portfolio
\ No newline at end of file
+# Portfolio: setup and mobile version skeleton
+
+# Project Description
+
+> For the second milestone in the process of creating your portfolio website, you will: <
+
+> set up a new repository and prepare it for development using best practices (e.g. linters). <
+> build the first 2 sections of the mobile website using the template you chose in the previous step. <
+
+> This project is about setting up a mobile version skeleton <
+
+# Screenshot
+ [screenshot](./images/screenshot.png)
+ [screenshot](./images/screenshot2.png)
+
+
+## Built with
+- HTML & CSS
+
+
+## Authors
+
+👤 **Author1**
+
+- GitHub: [@githubhandle](https://github.com/ger619)
+- Twitter: [@twitterhandle](https://twitter.com/ger_abol)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/david-ger-426b4576/)
```

Comment:
Rather than saying `@githubhandle`, you can replace it with your name like `@David` the same applies to `Twitter` and `LinkedIn`.

Reply:
Done